reviewdog.go

```diff
@@ -35,6 +39,8 @@ type CheckResult struct {
 	Lines   []string // Original error lines (often one line)
 }
 
+var vetError = fmt.Sprintf("", 1, 2, 3) // TODO(#20): revert
```

Comment:
:dog:  [[reviewdog](https://github.com/haya14busa/reviewdog)] :dog:
no formatting directive in Sprintf call